docs/Acceptable-Casks.md

```diff
@@ -26,13 +26,9 @@ We maintain separate taps for different types of binaries. Our nomenclature is:
 
 Stable versions live in the main repository at [Homebrew/homebrew-cask](https://github.com/Homebrew/homebrew-cask). They should run on the latest release of macOS or the previous point release (Monterey and Ventura as of late 2022).
 
-#### But there is no Stable version
-
-When software is only available as a beta, development, or unstable version, its cask can go in the main `homebrew/cask` repository. When stable versions become available, only those will be accepted as subsequent updates.
```

Comment:
We may want to restore some variant of this, e.g.

* If a new Cask `foo` only has `beta` releases, then it can use the `foo` token.
* When Cask goes stable, `foo` should refer to stable release
* If software maintains a beta release series, then `foo@beta` can be introduced if it meets our requirements.

This matches existing Casks in `homebrew-cask` repo and avoids introducing a `foo@beta` without a corresponding `foo` Cask.

Reply:
This would make sense to me 👍🏻. Alternatively I can see value in keeping `@beta` around here indefinitely to differentiate from Homebrew/homebrew-core that doesn't accept beta versions by default. Can you open a PR @cho-m either way?